app/lib/features/search/widgets/quick_actions_builder.dart

```diff
@@ -72,7 +72,10 @@ class QuickActionsBuilder extends ConsumerWidget {
           canPostPin
               ? OutlinedButton.icon(
                   key: QuickJumpKeys.createPinAction,
-                  onPressed: () => context.pushNamed(Routes.actionAddPin.name),
+                  onPressed: () => navigateTo(
+                    route: Routes.actionAddPin,
+                    push: true,
```

Comment:
care to elaborate? this entire actions routing things is kinda fragile and frickely and changes like this can have unexpected consequences ... did you run the UI test-suite after doing this change? Does it still pass?

Reply:
reason is to to keep consistency .i.e. as I see many widgets here are using mixed up navigation methods so its better to have actions routing be confined to ```navigateTo``` only. I haven't ran the test-suite but tested a lot during PR work by navigating in-between a lot and seems no issue.